config/confighttp/confighttp.go

```diff
@@ -476,6 +476,11 @@ type CORSConfig struct {
 	MaxAge int `mapstructure:"max_age"`
 }
 
+// NewDefaultCORSConfig creates a default cross-origin resource sharing (CORS) configuration.
+func NewDefaultCORSConfig() *CORSConfig {
```

Comment:
We should use this in `NewDefaultServerConfig`

Reply:
If you merge this, I promise to open a PR to do that

Reply:
`NewDefaultServerConfig` already exists in `confighttp` right? Can we add it now?

Reply:
yes, done.